readthedocs/projects/models.py

```diff
@@ -994,9 +1002,13 @@ def add_features(sender, **kwargs):
     # Feature constants - this is not a exhaustive list of features, features
     # may be added by other packages
     USE_SPHINX_LATEST = 'use_sphinx_latest'
+    USE_SETUPTOOLS_LATEST = 'use_setuptools_latest'
+    ALLOW_DEPRECATED_WEBHOOKS = 'allow_deprecated_webhooks'
```

Comment:
Are we not defining their type here, instead of in the migration? I don't know how I'd notice that these are "default true by date" without looking at the actual DB?

Reply:
Not sure there is great way to do that.

For instance, this feature might initially start evaluating True for historical projects and for explicitly tagged projects, and the migration is to ensure this happens. The next step with this feature is to explicitly add the historical projects actually using the feature and disallow all projects (default to False historically). So this meaning will mutate with time.

Besides documenting here, is there any pattern you think would work?

Reply:
My above question is probably more useful -- can I look in the admin to see what the properties are? If so, "the data in the DB" can be canonical.

Reply:
Correct, these options will be in the admin if any projects are assigned. Details on date added and historical value are in the admin detail page:

![image](https://user-images.githubusercontent.com/1140183/32240013-c3f98c46-be31-11e7-95ee-9e22c276cd00.png)